src/main/webapp/app/course/manage/detail/course-detail.component.html

```diff
@@ -60,6 +60,12 @@ <h2><span jhiTranslate="artemisApp.course.detail.title">Course Details:</span></
             <dd id="course-short-name">
                 <span>{{ course.shortName }}</span>
             </dd>
+            <ng-container *ngIf="!!course.organizations?.length">
```

Comment:
not necessary for `ngIf`. (optional)
```suggestion
            <ng-container *ngIf="course.organizations?.length">
```

Reply:
Alright, thanks, I will have a future PR for the course details again and will fix it there.